plugin/storage/integration/elasticsearch_test.go

```
@@ -100,53 +101,67 @@ func (s *ESStorageIntegration) getVersion() (uint, error) {
 	return uint(esVersion), nil
 }
 
-func (s *ESStorageIntegration) initializeES(allTagsAsFields, archive bool) error {
+func (s *ESStorageIntegration) initializeES(t *testing.T, allTagsAsFields, archive bool) error {
 	rawClient, err := elastic.NewClient(
 		elastic.SetURL(queryURL),
 		elastic.SetSniff(false))
-	if err != nil {
-		return err
-	}
+	require.NoError(t, err)
 	s.logger, _ = testutils.NewLogger()
 
 	s.client = rawClient
 	s.v8Client, err = elasticsearch8.NewClient(elasticsearch8.Config{
 		Addresses:            []string{queryURL},
 		DiscoverNodesOnStart: false,
 	})
-	if err != nil {
-		return err
-	}
+	require.NoError(t, err)
+
+	s.initSpanstore(t, allTagsAsFields, archive)
+	s.initSamplingStore(t)
 
-	if err := s.initSpanstore(allTagsAsFields, archive); err != nil {
-		return err
-	}
 	s.CleanUp = func() error {
-		return s.esCleanUp(allTagsAsFields, archive)
+		return s.esCleanUp(t, allTagsAsFields, archive)
 	}
 	s.Refresh = s.esRefresh
-	s.esCleanUp(allTagsAsFields, archive)
+	s.esCleanUp(t, allTagsAsFields, archive)
 	// TODO: remove this flag after ES support returning spanKind when get operations
 	s.GetOperationsMissingSpanKind = true
 	return nil
 }
 
-func (s *ESStorageIntegration) esCleanUp(allTagsAsFields, archive bool) error {
+func (s *ESStorageIntegration) esCleanUp(t *testing.T, allTagsAsFields, archive bool) error {
 	_, err := s.client.DeleteIndex("*").Do(context.Background())
-	if err != nil {
-		return err
-	}
-	return s.initSpanstore(allTagsAsFields, archive)
+	require.NoError(t, err)
+	return s.initSpanstore(t, allTagsAsFields, archive)
```

don't return, check (e.g. error is not checked in L125)